src/cascadia/TerminalSettingsModel/KeyMapping.idl

```diff
@@ -13,7 +13,6 @@ namespace Microsoft.Terminal.Settings.Model
         OpenNewTabDropdown,
         DuplicateTab,
         NewTab,
-        NewWindow,

```

Comment:
Looks like you just moved around these enums. Could probably revert changes here.

Reply:
FYI I intentionally moved them all to be at the end so that the order was more consistent, and reflective of the order these things were added in. `NewWindow` already existing in the list was maybe an oversight? I suppose I reserved the key long before it ever was used?